plugins/rest/help.md

```diff
@@ -248,17 +246,17 @@ This action is used to make a PATCH request.
 
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
-|body|object|None|False|Payload to submit to the server when making the HTTP Request call|None|{"user": "user@example.com"}|
+|body|[]object|None|False|Payload to submit to the server when making the HTTP Request call|None|{"user": "user@example.com"}|
```

Comment:
It seems that it didn't catch example in here 😃 
```suggestion
|body|[]object|None|False|Payload to submit to the server when making the HTTP Request call|None|[{"user": "user@example.com"}]|
```

Reply:
It will affect previous workflows so I have updated the version to 6.0.0 as the previous workflows will need to update their inputs